index_rs/src/index_rs/tests/tests.sh

```diff
@@ -6,9 +6,11 @@
 // * adapt the canister IDs defined below to the ones that were deployed
 // * then run % ic-repl tests.sh
 
-import IndexCanister = "rrkah-fqaaa-aaaaa-aaaaq-cai";
+import IndexCanister = "q3fc5-haaaa-aaaaa-aaahq-cai";
+import ClientCanister = "qhbym-qaaaa-aaaaa-aaafq-cai";
```

Comment:
Now we have the client canister ID twice, but I guess that's fine.

Reply:
Hm yeah that's ugly, but I can't manage to extract a variable / constant for it. :-(